clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -108,6 +108,8 @@
     UNREGISTER_BROKER(ApiMessageType.UNREGISTER_BROKER, false, RecordBatch.MAGIC_VALUE_V0, true),
     DESCRIBE_TRANSACTIONS(ApiMessageType.DESCRIBE_TRANSACTIONS),
     LIST_TRANSACTIONS(ApiMessageType.LIST_TRANSACTIONS),
+    GET_TELEMETRY_SUBSCRIPTIONS(ApiMessageType.GET_TELEMETRY_SUBSCRIPTIONS, true, true),
```

Comment:
I know there was another comment about the plural naming of this RPC, but the current spec is singular, not plural, because it only takes a single instance ID. `List*` and `Describe*` APIs are plural because they deal with multiple entities at once, but this call is more analogous to `InitProducerId` or `JoinGroup`: it's issued by a single client in order to coordinate its state with the broker. We may want List and Describe APIs for telemetry subscriptions as well (either now or in the future), but that isn't what this API is supposed to be, right? Or have I misunderstood?

Reply:
@bob-barrett  there could be multiple subscriptions qualified for the given client  and send to the client. So I guess plural is fine here.